apps/client/src/app/core/data/localized-data.service.ts

```diff
@@ -292,6 +227,47 @@ export class LocalizedDataService {
     return array[id];
   }
 
+  private getRowWithExtendedLanguage(key: keyof LazyData, id: number | string): I18nName {
+    let row = this.getRow<I18nName>(this.lazyData.data[key], id);
```

Comment:
Because row is never reassigned, you should use `const` instead of `let` (this breaks linter in CI).
```suggestion
    const row = this.getRow<I18nName>(this.lazyData.data[key], id);
```